whitepaper/cactus_arch.svg

```diff
@@ -0,0 +1,1777 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@sfuji822 

I added a boundary around the validators wrapping them with the API server blue box as well and added a legend entry for the same thing. This is is roughly what I was talking about during our last call.

![C: Users shingo_fujimoto Desktop cactus_arch2020-10-26](https://user-images.githubusercontent.com/1583231/99292911-bd854d00-27f6-11eb-8c66-8e7c1c5bd898.png)

Reply:
Humm. That is something different from my understanding.
The icon in the Validator was Ledger, which may consist of multiple nodes that participated in the blockchain network. 
However, Validator (API) Server will be the point of contact for the Ledger, and it will be an independent server from other Validators.
This is because Validator is a client or node of Ledger which is operated by the BLP of CACTUS.

Reply:
> Humm. That is something different from my understanding.
> The icon in the Validator was Ledger, which may consist of multiple nodes that participated in the blockchain network.
> However, Validator (API) Server will be the point of contact for the Ledger, and it will be an independent server from other Validators.
> This is because Validator is a client or node of Ledger which is operated by the BLP of CACTUS.

@sfuji822 Gotcha, I'll take a step back/zoom out a bit in that case: The validator. => Is the logic of the validator implemented as a plugin or is the logic prescribed/hardcoded?